### DISCUSSION TITLE
essay/2022/11/22/small-market

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#131] [Comments]
# essay/2022/11/22/small-market

투자자가 창업자에게 묻는 식상한 질문들. “그래서 시장 규모가 얼마나 되는데요?” 창업자는 뻥을 좀 보태 시장규모를 크게 말합니다. 투자자는 또 물어봅니다. “그거 네이버나 카카오가 따라 하면 어쩔 건데요?”

https://jeho.page/essay/2022/11/22/small-market.html

**Comment by GoBeromsu** [comment]
좋은 글 감사합니다 즐거운 추석 되십쇼!

**Reply by BenjaminKim** [reply]
추석 잘 보내세요!
